modules/network/virtual-network/version.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
+  "version": "0.1",
```

Comment:
Why version **0.1**? In the ResourceModules repo it's 0.4. Keeping the versions in sync with the source might help us in the long run.

Reply:
I think let's discuss this with the team and agree on how we want to start on the version. I set it as 0.1 because it's our first version.

Reply:
Updated to 0.4